src/main/java/org/broadinstitute/hellbender/utils/pileup/ReadPileup.java

```diff
@@ -0,0 +1,392 @@
+package org.broadinstitute.hellbender.utils.pileup;
+
+import com.google.api.client.repackaged.com.google.common.annotations.VisibleForTesting;
```

Comment:
Import `com.google.common.annotations.VisibleForTesting` instead of `com.google.api.client.repackaged.com.google.common.annotations.VisibleForTesting`